src/state/opamSysInteract.ml

```diff
@@ -429,12 +429,18 @@ let packages_status packages =
        >bmake
     *)
     let sys_installed =
-      run_query_command "brew" ["list"; "--formula"]
+      run_query_command "brew" ["list"; "--full-name"]
```

Comment:
Can you add/update the example output above

Reply:
it is up-to-date

Reply:
ok, thinking about it maybe i can add the `path/to/cask` form too

Reply:
Done